sandbox/inventory-datacloud.yaml

```diff
@@ -5,5 +5,5 @@ datacloud:
       # Change the host and the port accordingly
       ansible_host: localhost
       ansible_port: 22
-      ansible_user: ubuntu
+      ansible_user: azureuser
```

Comment:
I think the default user could be `ubuntu` because it's the default in the Ubuntu Cloud Images.

Reply:
Makes sense. I updated to ubuntu as default in the inventory file and the README installation instructions for sandbox.